lib/segment/src/index/field_index/field_index_base.rs

```diff
@@ -373,8 +373,8 @@ impl FieldIndex {
         match self {
             FieldIndex::KeywordIndex(index) => Some(FacetIndex::Keyword(index)),
             FieldIndex::IntMapIndex(index) => Some(FacetIndex::Int(index)),
-            | FieldIndex::UuidMapIndex(_) // TODO(facets): use uuid index too
-            | FieldIndex::UuidIndex(_)
+            FieldIndex::UuidMapIndex(index) => Some(FacetIndex::Uuid(index)),
```

Comment:
that's unrelated but on purpose?

Reply:
It is related only in the "Things I forgot to do for facets" thematic